README.md

```diff
@@ -3,7 +3,7 @@
 
 Change toolchains.xml to match your Java 1.8 and Java 11 installation locations, the current should work if your using a Mac with Java 1.8.0 update 192 and Java 11.0.5.
 ```
-$ /mvnw -t toolchains.xml clean install
+$ /mvnw -t toolchains.xml clean verify
```

Comment:
if the build fails and maven suggests using resume from, that will fail as the previous modules won't exist, that is why i do install.
also some IDE's use the installed jar files, so if you want to open the project in an IDE it will fail as dependencies won't exist

Reply:
we're not talking about a multimodule project, so resume-from doesn't make sense.